pkg/services/dashboards/dashboard_service.go

```diff
@@ -275,6 +291,16 @@ func (s *FakeDashboardService) ImportDashboard(dto *SaveDashboardDTO) (*models.D
 	return s.SaveDashboard(dto)
 }
 
+func (s *FakeDashboardService) DeleteDashboard(dashboardId int64, orgId int64) error {
```

Comment:
Can be removed now, right?

Reply:
Not anymore as I added `DeleteDashboard` function with the provisioned validation in the refactor.